tools/istio-iptables/pkg/capture/run.go

```diff
@@ -749,20 +750,28 @@ func (cfg *IptablesConfigurator) executeIptablesRestoreCommand(isIpv4 bool) {
 
 	log.Infof("Running %s with the following input:\n%v", cmd, strings.TrimSpace(data))
 	// --noflush to prevent flushing/deleting previous contents from table
-	cfg.ext.RunOrFail(cmd, strings.NewReader(data), "--noflush")
+	return cfg.ext.Run(cmd, strings.NewReader(data), "--noflush")
 }
 
-func (cfg *IptablesConfigurator) executeCommands() {
+func (cfg *IptablesConfigurator) executeCommands() error {
 	if cfg.cfg.RestoreFormat {
 		// Execute iptables-restore
-		cfg.executeIptablesRestoreCommand(true)
+		if err := cfg.executeIptablesRestoreCommand(true); err != nil {
```

Comment:
nit: running the two iptablesRestores for ipv4 and ipv6 could probably be refactored into a helper function